### DISCUSSION TITLE
Status of the project

### DISCUSSION THREAD

**Discussion by kalekseev** [palantir/blueprint#6768] [General]
Hi @ericjeney @invliD looks like @adidahiya has left the project and there has been no active development for the last two months. For example, there is a nasty bug https://github.com/palantir/blueprint/issues/6752, which was not addressed in any way for two weeks. Can you provide us with any details about the future of the project, can we expect the same level of support as before in the near future?

**Comment by ericjeney** [comment]
Hey @kalekseev! Thanks for reaching out. 🙂 

We’re committed to keeping this project staffed, and are working on finding a new full time maintainer. In the meantime, we'll be a bit slower responding to issues and pull requests, but I expect us to return to normal in the coming months.

For that issue in particular, we'll take a look this week.

**Comment by Lonli-Lokli** [comment]
I also interested in v6, is it even planned now?

**Comment by ggdouglas** [comment]
Closing this out since the situation has changed quite a bit since 2024.

Blueprint v6 shipped, and the project is actively maintained again. We have a few contributors keeping things moving: @ggdouglas, @mm-wang, and @cameronjoyner.

If you run into specific issues today, please open a fresh issue and we'll take a look.
